lms/templates/fields/field_image.underscore

```diff
@@ -1,17 +1,17 @@
 <div class="image-wrapper">
-    <img class="image-frame" src="<%- imageUrl %>"  alt="<%=imageAltText%>"/>
+    <img class="image-frame" src="<%- imageUrl %>"  alt="<%= HtmlUtils.ensureHtml(imageAltText) %>"/>
```

Comment:
Again, would it be better to have the code for the text directly in the template and drop all of these ensureHtml() calls?